composer.json

```diff
@@ -63,6 +63,7 @@
         "jms/di-extra-bundle": "^1.9",
         "jms/translation-bundle": "^1.4",
         "matthiasnoback/symfony-dependency-injection-test": "^3.1",
+        "phpspec/prophecy": "1.8.*",
```

Comment:
This change should be done in a separate PR, and the constraint should trust semver.

Reply:
Yes please use ^1.8